dbt version updates (#9590)
fishtown-analytics/docs.getdbt.com: website/snippets/_fusion-troubleshooting.md

If the extension reports that the <Constant name="fusion_engine" /> binary can't be found at the configured path (for example, `dbt-fusion binary not found at [path]`), the `dbt.fusionPath` setting is pointing to a location that doesn't contain a valid binary.

- Verify that [`dbt.fusionPath`](/docs/configure-dbt-extension#dbt-extension-settings) points to a valid <Constant name="fusion" /> binary.
- If you haven't installed <Constant name="fusion" /> manually, clear the setting and let the extension download and manage it for you.
- To install manually, follow [Install the <Constant name="fusion" /> CLI](/docs/local/install-dbt?version=2).
</Expandable>

<Expandable alt_header="dbt Fusion version is not compatible with this extension">

If the extension reports that the installed <Constant name="fusion" /> version isn't compatible with your dbt VS Code extension version, the two are outside the supported range.

1. Run `dbt --version` to check your installed <Constant name="fusion" /> version.
2. Compare it against the [version compatibility matrix](/docs/fusion/fusion-version-compatibility#compatibility-matrix) for your extension version.

> 3. Update <Constant name="fusion" /> or the extension so both fall within the supported range. Use the **Download compatible version** action in the notification if it's shown.

Reviewer: ```suggestion
3. Update <Constant name="fusion" /> or the extension so both fall within the supported range. Use the **Download compatible version** action in the notification if it appears.
```

</Expandable>

<Expandable alt_header="dbt Fusion crashes on startup">

> If the extension reports that <Constant name="fusion" /> crashed on startup, first confirm the binary runs on its own:

Reviewer: ```suggestion
If the extension reports that <Constant name="fusion" /> crashed on startup, confirm the binary runs on its own:
```

1. Run `dbt --version` in your terminal. If this fails, reinstall <Constant name="fusion" /> using [Install the <Constant name="fusion" /> CLI](/docs/local/install-dbt?version=2).
2. Use the **Show Logs** action in the notification (or open the **Output** tab) to review the startup error.
</Expandable>

<Expandable alt_header="A known-bad dbt Fusion version is installed">

> If the extension warns that your installed <Constant name="fusion" /> version has a known regression, dbt Labs has flagged that release as [known-bad](/docs/fusion/fusion-version-compatibility#known-bad-releases). Update to the version named in the notification:

Reviewer: ```suggestion
If the extension warns that your installed <Constant name="fusion" /> version has a known regression, dbt Labs has flagged that release as [known-bad](/docs/fusion/fusion-version-compatibility#known-bad-releases). Update to the version named in the notification.
```
